### DISCUSSION TITLE
Why are we avoiding anthropomorphism?

### DISCUSSION THREAD

**Discussion by Algoroxyolo** [sotopia-lab/sotopia#38] [Q&A] [answered]
Hi! I am Lorenzo. Me among a few others were working on roleplay agents-where our goal is to make the agents performing more like the character themselves.

Reading Sotopia leaves me puzzled, as across the paper the work has been constantly avoiding anthropomorphism. Can you elaborate the reason why?

thank you so much!

**Comment by XuhuiZhou** [comment, marked as answer]
Yeah, check this? https://arxiv.org/pdf/2402.02056.pdf
